pkg/cmd/task/logs.go

```diff
@@ -0,0 +1,196 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/cmd/taskrun"
+	"github.com/tektoncd/cli/pkg/flags"
+	"github.com/tektoncd/cli/pkg/helper/options"
+	thelper "github.com/tektoncd/cli/pkg/helper/task"
+	trlist "github.com/tektoncd/cli/pkg/helper/taskrun/list"
+	validate "github.com/tektoncd/cli/pkg/helper/validate"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func nameArg(args []string, p cli.Params) error {
+	if len(args) == 1 {
+		c, err := p.Clients()
+		if err != nil {
+			return err
+		}
+		name, ns := args[0], p.Namespace()
+		if _, err = c.Tekton.TektonV1alpha1().Tasks(ns).Get(name, metav1.GetOptions{}); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func logCommand(p cli.Params) *cobra.Command {
+	opts := options.NewLogOptions(p)
+
+	eg := `
+  # interactive mode: shows logs of the selected task run
+    tkn task logs -n namespace
+
+  # interactive mode: shows logs of the selected taskrun of the given task
+    tkn task logs task -n namespace
+
+  # show logs of given task for last run
+    tkn task logs task -n namespace --last
+
+  # show logs for given task and taskrun
+    tkn task logs task run -n namespace
```

Comment:
```suggestion
    tkn task logs taskrun -n namespace
```

Reply:
I'm assuming that you went with this because of what is in pipeline logs, so feel free to change it there as well. Otherwise, we can get it in a follow up pr.